### PR TITLE
Create cluster-intro.html

### PR DESCRIPTION
**Pull request by MashiatK**
Signed-off-by: Mashiat Jamal mashiatjamal91@gmail.com
Localizes content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html to Bengali
This is an initial PR as may need some improvements!
Tracking issue: #31677

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
please sign CLA

**Comment by utkarsh-singh1** [comment]
Hi @MashiatK, please follow these guides to sign easycla,

You can find about EasyCLA here - https://docs.linuxfoundation.org/lfx/easycla

and the steps to sign EasyCLA are here - https://docs.linuxfoundation.org/v2/easycla/contributors/individual-contributor

You can start your signing EasyCLA from here - https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/38056/#/?version=2

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: একটি ক্লাস্টার তৈরি করতে Minikube ব্যবহার করে
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:2]:
```suggestion
title: একটি ক্লাস্টার তৈরি করতে Minikube ব্যবহার করো
```

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                    <li>Kubernetes ক্লাস্টার কি তা জানুন.</li>
+                    <li>Minikube কি তা জানুন.</li>
+                    <li>একটি অনলাইন টার্মিনাল ব্যবহার করে একটি Kubernetes ক্লাস্টার শুরু করুন।</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3>Kubernetes Clusters</h3>
+                <p>
+                <b>কুবারনেটস কম্পিউটারের একটি অত্যন্ত উপলব্ধ ক্লাস্টার সমন্বয় করে যা একটি একক হিসাবে কাজ করার জন্য সংযুক্ত।</b> Kubernetes-এর বিমূর্ততা আপনাকে বিশেষভাবে পৃথক মেশিনে না বেঁধে একটি ক্লাস্টারে কন্টেইনারাইজড অ্যাপ্লিকেশন স্থাপন করতে দেয়। স্থাপনার এই নতুন মডেলটি ব্যবহার করার জন্য, অ্যাপ্লিকেশনগুলিকে এমনভাবে প্যাকেজ করা দরকার যাতে সেগুলিকে পৃথক হোস্ট থেকে আলাদা করা যায়: তাদের কন্টেইনারাইজ করা দরকার। কন্টেইনারাইজড অ্যাপ্লিকেশনগুলি অতীতের স্থাপনার মডেলগুলির তুলনায় আরও নমনীয় এবং উপলব্ধ, যেখানে অ্যাপ্লিকেশনগুলি সরাসরি হোস্টের সাথে গভীরভাবে একত্রিত প্যাকেজ হিসাবে নির্দিষ্ট মেশিনে ইনস্টল করা হয়েছিল। <b>Kubernetes একটি ক্লাস্টার জুড়ে অ্যাপ্লিকেশন কন্টেইনারগুলির বিতরণ এবং সময়সূচীকে আরও দক্ষ উপায়ে স্বয়ংক্রিয় করে।</b> Kubernetes একটি ওপেন সোর্স প্ল্যাটফর্ম এবং এটি উৎপাদনের জন্য প্রস্তুত।
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:32]:
```suggestion
                <b>কুবারনেটিস কম্পিউটারের একটি অত্যন্ত উপলব্ধ ক্লাস্টার সমন্বয় করে যা একটি একক হিসাবে কাজ করার জন্য সংযুক্ত।</b> Kubernetes-এর বিমূর্ততা আপনাকে বিশেষভাবে পৃথক মেশিনে না বেঁধে একটি ক্লাস্টারে কন্টেইনারাইজড অ্যাপ্লিকেশন স্থাপন করতে দেয়। স্থাপনার এই নতুন মডেলটি ব্যবহার করার জন্য, অ্যাপ্লিকেশনগুলিকে এমনভাবে প্যাকেজ করা দরকার যাতে সেগুলিকে পৃথক হোস্ট থেকে আলাদা করা যায়: তাদের কন্টেইনারাইজ করা দরকার। কন্টেইনারাইজড অ্যাপ্লিকেশনগুলি অতীতের স্থাপনার মডেলগুলির তুলনায় আরও নমনীয় এবং উপলব্ধ, যেখানে অ্যাপ্লিকেশনগুলি সরাসরি হোস্টের সাথে গভীরভাবে একত্রিত প্যাকেজ হিসাবে নির্দিষ্ট মেশিনে ইনস্টল করা হয়েছিল। <b>Kubernetes একটি ক্লাস্টার জুড়ে অ্যাপ্লিকেশন কন্টেইনারগুলির বিতরণ এবং সময়সূচীকে আরও দক্ষ উপায়ে স্বয়ংক্রিয় করে।</b> Kubernetes একটি ওপেন সোর্স প্ল্যাটফর্ম এবং এটি উৎপাদনের জন্য প্রস্তুত।
```

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                </p>
+                <p>>A কুবারনেটস ক্লাস্টার দুটি ধরণের সংস্থান নিয়ে গঠিত:
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:34]:
```suggestion
                <p>>একটি কুবারনেটিস ক্লাস্টার দুটি ধরণের সংস্থান নিয়ে গঠিত:
```

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+                    <ul>
+                        <li>The <b>Control Plane</b> coordinates the cluster</li>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:37]: Kindly translate these words `Control Plane`  , `coordinates the cluster`

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+                        <li><b>Nodes</b> are the workers that run applications</li>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:38]: Kindly translate these words too. `Nodes`, `are the workers that run applications`

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+                    </ul>
+                </p>
+            </div>
+
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Summary:</h3>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:45]: Kindly translate this word: `Summary:`

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+                    <ul>
+                        <li>Kubernetes cluster</li>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:47]: Kindly translate this word `Kubernetes cluster` , `Minikube`

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+                        <li>Minikube</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i>
+                         কুবারনেটস হল একটি প্রোডাকশন-গ্রেড, ওপেন-সোর্স প্ল্যাটফর্ম যা কম্পিউটার ক্লাস্টারের মধ্যে এবং জুড়ে অ্যাপ্লিকেশন কন্টেইনারগুলির প্লেসমেন্ট (শিডিউলিং) এবং এক্সিকিউশনকে অর্কেস্ট্রেট করে।
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:53]:
```suggestion
                         কুবারনেটিস হল একটি প্রোডাকশন-গ্রেড, ওপেন-সোর্স প্ল্যাটফর্ম যা কম্পিউটার ক্লাস্টারের মধ্যে এবং জুড়ে অ্যাপ্লিকেশন কন্টেইনারগুলির প্লেসমেন্ট (শিডিউলিং) এবং এক্সিকিউশনকে অর্কেস্ট্রেট করে।
```

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+                    </i></p>
+                </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2 style="color: #3771e3;">Cluster Diagram</h2>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:63]: Translate the `Cluster Diagram`

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_01_cluster.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><b>কন্ট্রোল প্লেন ক্লাস্টার পরিচালনার জন্য দায়ী.</b>কন্ট্রোল প্লেন আপনার ক্লাস্টারে সমস্ত ক্রিয়াকলাপ সমন্বয় করে, যেমন অ্যাপ্লিকেশনের সময় নির্ধারণ, অ্যাপ্লিকেশনগুলির পছন্দসই অবস্থা বজায় রাখা, অ্যাপ্লিকেশনগুলিকে স্কেলিং করা এবং নতুন আপডেটগুলি রোল আউট করা.</p>
+                <p><b>একটি নোড হল একটি VM বা একটি শারীরিক কম্পিউটার যা একটি কুবারনেটস ক্লাস্টারে কর্মী মেশিন হিসাবে কাজ করে।</b> প্রতিটি নোডের একটি কুবেলেট থাকে, যা নোড পরিচালনা এবং কুবারনেটস কন্ট্রোল প্লেনের সাথে যোগাযোগের জন্য একটি এজেন্ট। নোডের কনটেইনার ক্রিয়াকলাপ পরিচালনা করার জন্য সরঞ্জাম থাকা উচিত, যেমন কন্টেইনার বা ডকার। একটি Kubernetes ক্লাস্টার যা উত্পাদন ট্র্যাফিক পরিচালনা করে তার ন্যূনতম তিনটি নোড থাকা উচিত কারণ যদি একটি নোড নিচে চলে যায়, তাহলে একটি etcd সদস্য এবং একটি কন্ট্রোল প্লেন ইনস্ট্যান্স উভয়ই হারিয়ে যায় এবং রিডানডেন্সি আপস করা হয়। আপনি আরো কন্ট্রোল প্লেন নোড যোগ করে এই ঝুঁকি কমাতে পারেন।</p>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:77]:
```suggestion
                <p><b>একটি নোড হল একটি VM বা একটি শারীরিক কম্পিউটার যা একটি কুবারনেটিস ক্লাস্টারে কর্মী মেশিন হিসাবে কাজ করে।</b> প্রতিটি নোডের একটি কুবেলেট থাকে, যা নোড পরিচালনা এবং কুবারনেটিস কন্ট্রোল প্লেনের সাথে যোগাযোগের জন্য একটি এজেন্ট। নোডের কনটেইনার ক্রিয়াকলাপ পরিচালনা করার জন্য সরঞ্জাম থাকা উচিত, যেমন কন্টেইনার বা ডকার। একটি Kubernetes ক্লাস্টার যা উৎপাদন ট্র্যাফিক পরিচালনা করে তার ন্যূনতম তিনটি নোড থাকা উচিত কারণ যদি একটি নোড নিচে চলে যায়, তাহলে একটি etcd সদস্য এবং একটি কন্ট্রোল প্লেন ইনস্ট্যান্স উভয়ই হারিয়ে যায় এবং রিডানডেন্সি আপস করা হয়। আপনি আরো কন্ট্রোল প্লেন নোড যোগ করে এই ঝুঁকি কমাতে পারেন।</p>
```

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>Control Planes manage the cluster and the nodes that are used to host the running applications.</i></p>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:82]: Kindly translate this sentence  "Control Planes manage the cluster and the nodes that are used to host the running applications."

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+
+                <p>একটি Kubernetes ক্লাস্টার শারীরিক বা ভার্চুয়াল মেশিনে স্থাপন করা যেতে পারে। Kubernetes বিকাশের সাথে শুরু করতে, আপনি Minikube ব্যবহার করতে পারেন। Minikube হল একটি হালকা ওজনের Kubernetes বাস্তবায়ন যা আপনার স্থানীয় মেশিনে একটি VM তৈরি করে এবং শুধুমাত্র একটি নোড ধারণকারী একটি সাধারণ ক্লাস্টার স্থাপন করে। Minikube Linux, macOS এবং Windows সিস্টেমের জন্য উপলব্ধ। Minikube CLI আপনার ক্লাস্টারের সাথে কাজ করার জন্য প্রাথমিক বুটস্ট্র্যাপিং ক্রিয়াকলাপগুলি প্রদান করে, যার মধ্যে শুরু, থামানো, স্থিতি এবং মুছে ফেলা হয়। এই টিউটোরিয়ালের জন্য, তবে, আপনি মিনিকুবের আগে থেকে ইনস্টল করা একটি প্রদত্ত অনলাইন টার্মিনাল ব্যবহার করবেন৷</p>
+                <p>এখন যেহেতু আপনি জানেন কুবারনেটস কী, আসুন অনলাইন টিউটোরিয়ালটিতে যাই এবং আমাদের প্রথম ক্লাস্টার শুরু করি!</p>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:92]:
```suggestion
                <p>এখন যেহেতু আপনি জানেন কুবারনেটিস কী, আসুন অনলাইন টিউটোরিয়ালটিতে যাই এবং আমাদের প্রথম ক্লাস্টার শুরু করি!</p>
```

File: content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive/" role="button">Start Interactive Tutorial <span class="btn__next">›</span></a>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:100]: Translate this sentence "Start Interactive Tutorial"

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**